packages/smooth_app/lib/pages/product/nutrition_add_nutrient_button.dart

```diff
@@ -55,9 +54,14 @@ class NutritionAddNutrientButton extends StatelessWidget {
                 controller: nutritionTextController,
                 onChanged: (String? query) => setState(
                   () => filteredList = leftovers
-                      .where((OrderedNutrient item) =>
-                          removeDiacritics(item.name!).toLowerCase().contains(
-                              removeDiacritics(query!).toLowerCase().trim()))
+                      .where(
+                        (OrderedNutrient item) => item.name!
+                            .removeDiacritics()
+                            .toLowerCase()
+                            .contains(
+                              query!.removeDiacritics().toLowerCase().trim(),
+                            ),
```

Comment:
Shouldn't that be first `toLowerCase` and the `removeDiacritics`, like on the rest of the code?